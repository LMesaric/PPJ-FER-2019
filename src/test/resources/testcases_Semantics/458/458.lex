KR_INT 1 int
IDN 1 main
L_ZAGRADA 1 (
KR_VOID 1 void
D_ZAGRADA 1 )
L_VIT_ZAGRADA 1 {
KR_INT 2 int
IDN 2 j
TOCKAZAREZ 2 ;
KR_FOR 3 for
L_ZAGRADA 3 (
IDN 3 j
OP_PRIDRUZI 3 =
BROJ 3 0
TOCKAZAREZ 3 ;
IDN 3 j
OP_LT 3 <
BROJ 3 10
TOCKAZAREZ 3 ;
OP_INC 3 ++
IDN 3 j
D_ZAGRADA 3 )
L_VIT_ZAGRADA 3 {
KR_BREAK 4 break
TOCKAZAREZ 4 ;
D_VIT_ZAGRADA 5 }
KR_BREAK 6 break
TOCKAZAREZ 6 ;
KR_BREAK 7 break
TOCKAZAREZ 7 ;
KR_BREAK 8 break
TOCKAZAREZ 8 ;
KR_RETURN 9 return
BROJ 9 0
TOCKAZAREZ 9 ;
D_VIT_ZAGRADA 10 }
