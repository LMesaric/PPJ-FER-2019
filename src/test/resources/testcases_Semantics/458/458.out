<naredba_skoka> ::= KR_BREAK(6,break) TOCKAZAREZ(6,;)
